tests/common.go

```diff
@@ -7581,3 +7581,182 @@ func GetClusterProviders() []string {
 	}
 	return clusterProviders
 }
+
+func IsPoolOffline(n node.Node) (bool, error) {
```

Comment:
delete if not required

Reply:
function is being used to check 2 things , if the pod percentage is 90% full or pod is offline , both the cases function GetPoolUuidsWithStorageFull returns true , will keep it for now . will optimise the funciton in future